[2025-05-28T10:50:03.006+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data_etl.load_weather_task scheduled__2025-05-28T10:45:00+00:00 [queued]>
[2025-05-28T10:50:03.032+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data_etl.load_weather_task scheduled__2025-05-28T10:45:00+00:00 [queued]>
[2025-05-28T10:50:03.033+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-05-28T10:50:03.046+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): load_weather_task> on 2025-05-28 10:45:00+00:00
[2025-05-28T10:50:03.054+0000] {standard_task_runner.py:60} INFO - Started process 1495 to run task
[2025-05-28T10:50:03.058+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'weather_data_etl', 'load_weather_task', 'scheduled__2025-05-28T10:45:00+00:00', '--job-id', '54', '--raw', '--subdir', 'DAGS_FOLDER/weather.py', '--cfg-path', '/tmp/tmpknmih9wj']
[2025-05-28T10:50:03.062+0000] {standard_task_runner.py:88} INFO - Job 54: Subtask load_weather_task
[2025-05-28T10:50:03.120+0000] {task_command.py:423} INFO - Running <TaskInstance: weather_data_etl.load_weather_task scheduled__2025-05-28T10:45:00+00:00 [running]> on host f1bd559e8a8b
[2025-05-28T10:50:03.218+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_data_etl' AIRFLOW_CTX_TASK_ID='load_weather_task' AIRFLOW_CTX_EXECUTION_DATE='2025-05-28T10:45:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-28T10:45:00+00:00'
[2025-05-28T10:50:03.703+0000] {python.py:201} INFO - Done. Returned value was: Data cuaca berhasil disimpan. 5 rekaman baru, 30 total dalam sejarah
[2025-05-28T10:50:03.735+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=weather_data_etl, task_id=load_weather_task, execution_date=20250528T104500, start_date=20250528T105003, end_date=20250528T105003
[2025-05-28T10:50:03.760+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-05-28T10:50:03.777+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
